spark/src/main/scala/org/apache/comet/serde/QueryPlanSerde.scala

```diff
@@ -210,8 +210,29 @@ object QueryPlanSerde extends Logging with ShimQueryPlanSerde with CometExprShim
       expr match {
         case agg: AggregateExpression =>
           agg.aggregateFunction match {
-            case _: Min | _: Max | _: Count =>
+            case _: Count =>
               Some(agg)
+            case min: Min =>
+              if (minMaxDataTypeSupported(min.dataType)) {
+                Some(agg)
+              } else {
+                withInfo(windowExpr, "Unsupported datatype", expr)
```

Comment:
Can we put the data type in the error message?

Reply:
Fixed. Thanks